public/scripts/auroragame.js

```diff
@@ -131,5 +142,8 @@ requirejs([
             game.scale.startFullScreen();
         }
     }
+    function toggleMusic() {
+        sound.toggleSound("music");
+    }
```

Comment:
This could maybe be changed later to take a parameter if you want to mute the sound of the characters too or only the music.